docs/quickstart.rst

```diff
@@ -36,34 +35,24 @@ For more information about how these packages relate to each other, consult the
 Explore the Use Cases
 ---------------------
 
-The Use Case Gallery demonstrates real-world science applications of the
+`Pangeo Gallery`_ showcases real-world science applications of the
 Pangeo stack. The use cases are formatted as `Jupyter Notebooks`_, a
 self-describing computational document including text, code, and figures.
 A great way to get started is to find a use case that is similar to the
 analysis you want to do and modify it to suit your needs.
 
-.. toctree::
-  :maxdepth: 2
-
-  use_cases/index
-
 A great way for novices to learn how to use Pangeo is to start from one of these
-use cases and modify it for your needs. Many of the use cases are pre-loaded
-on the Pangeo deployments listed below.
+use cases and modify it for your needs.
 
-Try Out a Pangeo Deployment
+Try Out a Pangeo
 ---------------------------
 
-There are several places where you can access a Pangeo deployment, from which
-you can analyze large datasets in a scalable, reproducible way.
-
-.. toctree::
-   :maxdepth: 2
-
-   deployments
-
-If none of these meets your needs, you can create your own deployment by
-following the :ref:`setup-guides`.
+The best way to try out Pangeo is to explore one of the interactive
+Binders in `Pangeo Gallery`_.
+If you're interested in working in a cloud based environment, you can sign
+up for :ref:`cloud`.
```

Comment:
how will this render? ...you can sign up for `XYZ`.

Reply:
Pangeo Cloud.